docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,40 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is a side-car process that connects the Sommelier chain to the outside world. It provides an API for Strategy Providers to rebalance Cellars through the Sommelier validator set based on off-chain calculations. Additionally, it subsumes the functionality of [gorc](https://github.com/PeggyJV/gravity-bridge/tree/main/orchestrator/gorc) for managing the [Gravity bridge Orchestrator](https://github.com/PeggyJV/gravity-bridge/tree/main/orchestrator), and provides developer tools and features for developing and testing Cellars.
+
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Strategy Providers (SPs) send requests whenever they determine that the market has changed enough to warrant action. The request payload contains everything needed to make a *cork*: a signed combination of a cellar address and an ABI encoded contract call. When Steward receives a submission from the SP, it validates the target cellar address, builds a cork, signs it with the delegate key, and submits it to the [Cork module](https://github.com/peggyjv/sommelier/tree/main/x/cork) on chain.
+
+> :bulb:  Steward authorizes function calls both by only supporting certain Cellar functions in its API, and by validating the target cellar before forwarding a call.
+
+## Quickstart
+
+To start Steward, simply run
+
+```bash
+steward -c <config_toml_path> start
+```
+
+> :warning: If this is production and your node is running, don't forget to make sure the Orchestrator is also running! If you don't, you can be jailed and your submitted corks will not be processed.
+See the [Orchestrator Quickstart](./docs/03-TheOrchestrator.md#quickstart) section of the docs.
+
+## Running Steward as a server
+
+### TLS Certificates
+
+Before SPs can establish a connection with your `steward` server, you will need to generate a CA, a TLS certificate signed by this CA, and you'll need to the SP's client CA. These certificates must be TLS 1.3 compliant. The paths to these values and the key used to create your signed server certificate must be configured in the `[server]` table of your config file. Starting out, Sommelier will be running the only SP, and Steward will use our client CA cert by default. This CA cert can be found in the `tls/` directory of this repo. There is no need to set the `client_ca_cert_path` configuration field at this time.
```

Comment:
"and you'll need to the SP's client CA" needs to be updated to whatever was meant to be said here.

Reply:
PeggyJV is running the SP, Sommelier is a protocol and not an organization.

Reply:
Might be worth mentioning specifically that it doesn't need to be set because it will default to the PeggyJV cert included in the binary in this first version.